src/ApiCalls/getArticles.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const getArticles = () => {
+    return axios.get('https://nc-news-kgw0.onrender.com/api/articles')
+    .then((response) => {
+        return response.data
+    })
+    .catch((err) => {
+        return "ERROR"
```

Comment:
Consider how you might set some error state that would allow you to provide feedback to a user.